### ISSUE TITLE
Grinder inventory issues

### ISSUE THREAD

**Issue by Dnoxl** [DraconisCluster/DX-Issues#33]
tl; dr
Comps in Grinders Inv blinking as if transferred, not receiving entirety of ground comps

Full info
Initial report by user xSupeFly - ticket 50223:
Grinders are being funky when grinding weight not increasing the cargo volume same and when looking into grinder inventory its showing the components blinking like its transfering but its like all the time I got video which i will post later need to edit it to make it smaller size. [video](https://cdn.discordapp.com/attachments/1213315733086543902/1213434273189732412/Untitled_video_-_Made_with_Clipchamp_10.mp4?ex=65f575bb&is=65e300bb&hm=c10e3717bfe82ec3cecc6eb698aff6e1bb4319a536c84789c3913ac4659695d3&)

User further states:
and it does grind down blocks but it will not collect materials only sometimes basically you grind but you get like 40% materials bc i know i was getting the tech and some comps into cargo fixship etc didn't refresh cargo. 
;
However after replacing the grinders the behaviour seemed to return to a normal state.

**Comment by Synirrr** [comment]
Was this raised to Ash in the CoreSystems discord?

**Comment by Dnoxl** [comment]
> Was this raised to Ash in the CoreSystems discord?

Nope, didn't think about that

**Comment by draconb** [comment]
Inv size was changed a while ago to fix issues around this
